docs/generated/sql/bnf/stmt_block.bnf

```diff
@@ -83,6 +84,9 @@ explain_stmt ::=
 	'EXPLAIN' explainable_stmt
 	| 'EXPLAIN' '(' explain_option_list ')' explainable_stmt
 
+export_stmt ::=
+	'EXPORT' import_data_format '(' string_or_placeholder opt_with_options ')' select_stmt
```

Comment:
So what was the rationale for not having INTO here? (as in `EXPORT INTO CSV ...`)

Reply:
nothing in particular, just didn't seem to need it? does it read better with it?

Reply:
We already have `BACKUP ... TO` and `RESTORE ... FROM`. `EXPORT INTO` reads better to my ears.